apps/mobile/app/utils/crashReporting.ts

```diff
@@ -23,46 +23,45 @@
  *  This is where you put your crash reporting service initialization code to call in `./app/app.tsx`
  */
 export const initCrashReporting = () => {
-  // Sentry.init({
-  //   dsn: "YOUR DSN HERE",
-  //   enableInExpoDevelopment: true,
-  //   debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
-  // })
-  // Bugsnag.start("YOUR API KEY")
+	Sentry.init({
+		dsn: process.env.EXPO_PUBLIC_SENTRY_DSN,
+		enableInExpoDevelopment: true,
+		debug: false, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
+	})
+	// Bugsnag.start("YOUR API KEY")
 }
-
 /**
  * Error classifications used to sort errors on error reporting services.
  */
 export enum ErrorType {
-  /**
-   * An error that would normally cause a red screen in dev
-   * and force the user to sign out and restart.
-   */
-  FATAL = "Fatal",
-  /**
-   * An error caught by try/catch where defined using Reactotron.tron.error.
-   */
-  HANDLED = "Handled",
+	/**
+	 * An error that would normally cause a red screen in dev
+	 * and force the user to sign out and restart.
+	 */
+	FATAL = "Fatal",
+	/**
+	 * An error caught by try/catch where defined using Reactotron.tron.error.
+	 */
+	HANDLED = "Handled",
 }
 
 /**
  * Manually report a handled error.
  */
 export const reportCrash = (error: any, type: ErrorType = ErrorType.FATAL) => {
-  if (__DEV__) {
-    // Log to console and Reactotron in development
-    const message = error.message || "Unknown"
-    console.error(error)
-    console.log(message, type)
-    console.tron.log(error)
-  } else {
-    // In production, utilize crash reporting service of choice below:
-    // RN
-    // Sentry.captureException(error)
-    // Expo
-    // Sentry.Native.captureException(error)
-    // crashlytics().recordError(error)
-    // Bugsnag.notify(error)
-  }
+	if (__DEV__) {
+		// Log to console and Reactotron in development
+		const message = error.message || "Unknown"
+		console.error(error)
+		console.log(message, type)
+		console.tron.log(error)
+	} else {
+		// In production, utilize crash reporting service of choice below:
+		// RN
+		// Sentry.captureException(error)
+		// Expo
+		// Sentry.Native.captureException(error)
```

Comment:
@desperado1802 why you did not uncomment that line @desperado1802 ??? I.e. `Sentry.Native.captureException(error)`? That should work well I think!

Reply:
Sure, I will uncomment it in the next push. Wanted to be sure that the configuration so far is implemented as it should. Also as I'm not able to test it on Expo Go(It doesn't have Sentry SDK/ Native events)

Reply:
Did the changes you asked me to do @evereq. Did some documentation GitHub issues reading yesterday. There it's said that native events are not available for Expo Go. However in production and eas build should work. 

some of the references:
https://github.com/expo/sentry-expo/issues/361
https://docs.expo.dev/guides/using-sentry/

![image](https://github.com/ever-co/ever-teams/assets/124465103/a148b1e8-042f-4a3b-a4ca-a296874d80ca)